packages/web-react/src/components/Breadcrumbs/demo/BreadcrumbsCustom.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { Link } from '../../Link';
+import { Breadcrumbs } from '../Breadcrumbs';
+
+const BreadcrumbsCustom = () => {
+  const items = [
+    {
+      title: 'Root',
+      url: '#rootUrl',
+    },
+    {
+      title: 'Category',
+      url: '#categoryUrl',
+    },
+    {
+      title: 'Subcategory',
+      url: '#subcategoryUrl',
+    },
+    {
+      title: 'Current page',
+      url: '#currentUrl',
+    },
+  ];
+
+  return (
+    <Breadcrumbs goBackTitle="Back">
```

Comment:
Icon separators seem to be missing from this example:

![obrazek](https://github.com/lmc-eu/spirit-design-system/assets/5614085/004e1784-50b9-483c-baa0-485bd2b158f2)

Vs. `web`/`web-twig`:

![obrazek](https://github.com/lmc-eu/spirit-design-system/assets/5614085/bf4c7b3a-4930-47d8-931c-d5889285278c)

Reply:
Fixed, now it looks as it should